oddbird/Metecho
# Allow converting Project playground org into new Task dev org.

![](https://source.unsplash.com/featured/?cute,animal&210909)

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- Create a *Project* scratch org
- Make changes in the org
- Click "contribute" btn on org
- Select either "Create a new Epic and Task" or "Create a new Task with no Epic"
- Click "Contribute" btn
- If you selected "new epic and task":
  - fill out create epic form and submit
  - fill out create task form and submit
  - see the new task with your org as the Dev Org (and you assigned as the Dev), then the "retrieve changes" modal opens
- If you selected "new task with no epic":
  - fill out create task form and submit
  - see the new task with your org as the Dev Org (and you assigned as the Dev), then the "retrieve changes" modal opens

## Show me
_Provide screenshots/animated gifs/videos if necessary._

![Screen Shot 2021-09-09 at 12 36 28 PM](https://user-images.githubusercontent.com/552316/132726787-332bc856-9fd5-4721-9de8-fa5024870171.png)
![Screen Shot 2021-09-09 at 12 36 36 PM](https://user-images.githubusercontent.com/552316/132726791-ce159ce9-3154-492f-b0ec-9aa00121f4c1.png)
![Screen Shot 2021-09-09 at 12 36 42 PM](https://user-images.githubusercontent.com/552316/132726792-66c560a7-d515-469b-88a6-f8ad8ce5fcfc.png)
![Screen Shot 2021-09-09 at 12 36 53 PM](https://user-images.githubusercontent.com/552316/132726794-2ef54417-20cf-4147-a7ff-08a895ce2c3c.png)
